cmd/federation-controller/main.go

```diff
@@ -35,13 +43,6 @@ import (
 	"github.com/openshift-service-mesh/federation/internal/pkg/xds"
 	"github.com/openshift-service-mesh/federation/internal/pkg/xds/adsc"
 	"github.com/openshift-service-mesh/federation/internal/pkg/xds/adss"
-	routev1client "github.com/openshift/client-go/route/clientset/versioned"
-	istiokube "istio.io/istio/pkg/kube"
-	istiolog "istio.io/istio/pkg/log"
-	"istio.io/istio/pkg/slices"
-	corev1 "k8s.io/api/core/v1"
-	"k8s.io/client-go/informers"
-	"k8s.io/client-go/rest"
```

Comment:
@bartoszmajsak this is `fix-imports`, not me.

Reply:
I know, it's me. There's PR for that #103 so I won't mess it up again :)